<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushBtnGo">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>230</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Process</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonQuit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>230</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="pathToExiftool">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>321</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Path to exiftool</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>GPS Coordinates</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="gpsCoodinates">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>80</y>
      <width>281</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>sexagesimal/decimal longitude latitude [altitude]</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {background-color: rgb(254, 220, 220);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>64</number>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Picture(s)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnFiles">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>220</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Base folder</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="picturesFolder">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>120</y>
      <width>281</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnFolder">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>120</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="pictures">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>411</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame {background-color: rgb(254, 220, 220);}</string>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushClearFiles">
    <property name="geometry">
     <rect>
      <x>399</x>
      <y>220</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>181</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Website for coordinates</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="coordinateWebsite">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>281</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Noto Sans</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>https://www.daftlogic.com/sandbox-google-maps-find-altitude.htm</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushGotoWebsite">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Go</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkOverwrite">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Avoids creating &quot;original&quot; backup file(s)</string>
    </property>
    <property name="text">
     <string>Overwrite original</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBtnGetCoordFromFile">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>pushBtnGo</tabstop>
  <tabstop>pushButtonQuit</tabstop>
  <tabstop>pathToExiftool</tabstop>
  <tabstop>gpsCoodinates</tabstop>
  <tabstop>pushBtnFiles</tabstop>
  <tabstop>picturesFolder</tabstop>
  <tabstop>pushBtnFolder</tabstop>
  <tabstop>pictures</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
